src/Microsoft.DotNet.XHarness.Android/AdbRunner.cs

```diff
@@ -458,7 +458,7 @@ public List<string> PullFiles(string devicePath, string localPath)
             }
             catch (Exception toLog)
             {
-                logger.LogError(toLog, $"Exception thrown while trying to find compatible device with {propertyName} {apkRequiredProperty}");
+                logger.LogError(toLog, $"Exception thrown while trying to find compatible device with {propertyName} { string.Join("', '", apkRequiredProperty) }");

```

Comment:
nit: I guess you want `'` around as well if it's in between?

```suggestion
                logger.LogError(toLog, $"Exception thrown while trying to find compatible device with {propertyName} '{ string.Join("', '", apkRequiredProperty) }'");

```

Reply:
yep, thanks